src/lib/stores/contractStore.js

```diff
@@ -0,0 +1,33 @@
+import { writable } from 'svelte/store';
+import  Contract  from 'stellar-sdk';
+
+function createContractStore() {
+    const { subscribe, set, update } = writable({
+        savedContracts: loadSavedContracts()
+    });
+
+    return {
+        subscribe,
+        saveContract: (contract) => update(store => {
+            const newSavedContracts = [...store.savedContracts, contract];
+            localStorage.setItem('savedContracts', JSON.stringify(newSavedContracts));
+            return { ...store, savedContracts: newSavedContracts };
+        }),
+        removeContract: (contractId) => update(store => {
+            const newSavedContracts = store.savedContracts.filter(c => c.id !== contractId);
+            localStorage.setItem('savedContracts', JSON.stringify(newSavedContracts));
```

Comment:
Would love to see you take advantage of the `svelte-local-storage-store` package (which has since been renamed to `svelte-persisted-store`, but that change is out-of-scope for this PR). We're already using it in the `contactsStore`, so you could probably take some inspiration from there.

Reply:
used as indicated ser